control-agent/text-to-speech/moveFB.py

```diff
@@ -0,0 +1,20 @@
+import stretch_body.robot
+import time
```

Comment:
Did you pylint this? Maybe check the import order

Reply:
reordered